tests/e2e/analyze_graph/optimize_indexes.py

```diff
@@ -62,7 +62,7 @@ def test_analyze_graph_delete_statistics(delete_query, multi_db):
     # After deleting statistics, id2 should be chosen because it has less vertices
     expected_explain_after_delete_analysis = [
         (f" * Produce {{n}}",),
-        (f" * Filter (n :Label), {{n.id1}}, {{n.id2}}",),
+        (f" * Filter {{n.id1}}",),
```

Comment:
Did stuff change only when using hints or in both situations?

Reply:
in every situation of explain and profile

Reply:
which has multiple filtering operations